pkg/pillar/cmd/loguploader/loguploader.go

```diff
@@ -421,10 +421,15 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject, ar
 
 func sendCtxInit(ctx *loguploaderContext) {
 	//get server name
-	bytes, err := os.ReadFile(types.ServerFileName)
-	if err != nil {
-		log.Fatalf("sendCtxInit: Failed to read ServerFileName(%s). Err: %s",
-			types.ServerFileName, err)
+	var bytes []byte
+	var err error
+	for len(bytes) == 0 {
```

Comment:
Same concern as above: do we have an infinite loop here if file is empty?

Reply:
Same.

Reply:
Fixed